src/sentry/static/sentry/less/group-detail.less

```diff
@@ -1691,13 +1691,6 @@ span.val {
   margin: -2px -4px;
 }
 
-.val-string-multiline {
```

Comment:
Does this still need to be used for multi line strings if `valueInfo.isMultiLine` is `true`?

Reply:
Good catch - I'll remove this

Reply:
Note: This was adding an unnecessary background color and margin.